.studiorc

```diff
@@ -173,7 +173,7 @@ _load-if-not-loaded() {
 load-if-not-loaded() { stop-on-failure _load-if-not-loaded "$@"; }
 
 start-datastore() {
-  if hab sup status habitat/builder-datastore > /dev/null; then
+if hab sup status habitat/builder-datastore | grep "builder-datastore" > /dev/null; then
```

Comment:
Should this be `grep "habitat/builder-datastore"`?

Reply:
Just "builder-datastore" seems sufficient?

Reply:
My point was just to make the grepping consistent, given that on line 167, the grep will resolve to something like `grep habitat/builder-router` or whatever the package name is.